### ISSUE TITLE
[Feature Request][Tabler] choose default layout

### ISSUE THREAD

**Issue by tabacitu** [Laravel-Backpack/theme-tabler#9]
# Feature Request

### What's the feature you think Backpack should have?

Each v6 theme has a few layouts. What I want to do is be able to choose the default layout, in a config, to quickly switch between:

<img width="1512" alt="CleanShot 2023-03-23 at 12 26 52@2x" src="https://user-images.githubusercontent.com/1032474/227175330-915bd413-1974-4864-aeda-6cd22dcae63e.png">
<img width="1512" alt="CleanShot 2023-03-23 at 12 27 04@2x" src="https://user-images.githubusercontent.com/1032474/227175402-29112ea8-d7e1-4dba-8de5-0961c3480b42.png">
<img width="1512" alt="CleanShot 2023-03-23 at 12 27 17@2x" src="https://user-images.githubusercontent.com/1032474/227175474-0da0ccdd-bdcd-406c-8128-b19b8a9b3321.png">
<img width="1512" alt="CleanShot 2023-03-23 at 12 27 38@2x" src="https://user-images.githubusercontent.com/1032474/227175558-71204265-0dee-457e-92bb-53478ee6cd7a.png">
<img width="1512" alt="CleanShot 2023-03-23 at 12 27 53@2x" src="https://user-images.githubusercontent.com/1032474/227175624-31e63fbc-a8e4-4a19-934d-e471fef392e9.png">
<img width="1512" alt="CleanShot 2023-03-23 at 12 28 12@2x" src="https://user-images.githubusercontent.com/1032474/227175698-a6edf864-7436-4757-9c2b-69e739d46124.png">
<img width="1512" alt="CleanShot 2023-03-23 at 12 28 36@2x" src="https://user-images.githubusercontent.com/1032474/227175824-8e8d69b5-0439-43e8-b21f-939b74eff9ce.png">

Basically most (if not all) of the layouts that Tabler provides:
<img width="1512" alt="CleanShot 2023-03-23 at 12 28 50@2x" src="https://user-images.githubusercontent.com/1032474/227175951-92ab2500-320c-4a03-8055-66527487d777.png">

### Do you see this as a core feature, a theme feature or an add-on?

I'm... not sure. I guess it would be a theme feature? Something that only the `tabler` theme provides? I think we can code it like that, and if we think it should be more general... we move the wiring upstream...
